src/IO/WriteHelpers.h

```diff
@@ -998,4 +998,24 @@ inline String toString(const T & x)
     return buf.str();
 }
 
+inline void writeNullTerminatedString(const String & s, WriteBuffer & buffer)
+{
+    buffer.write(s.data(), s.size());
+    buffer.write(0);
+}
+
+template <typename T>
+inline std::enable_if_t<is_arithmetic_v<T> && (sizeof(T) <= 8), void>
+writeBinaryBigEndian(T x, WriteBuffer & buf)    /// Assuming little endian architecture.
```

Comment:
`boost::endian::native_to_big()` looks better and it requires no such assumptions.

Reply:
Ok.

Reply:
But it's Ok nevertheless :)